package.json

```diff
@@ -44,5 +44,12 @@
     "deploy-docs": "gh-pages -d docs",
     "preversion": "yarn build",
     "postpublish": "yarn build-docs && yarn deploy-docs"
+  },
+  "prettier": {        
+    "tabWidth": 2,
+    "semi": false,
```

Comment:
I typically leave semi on in my projects (`singleQuote` is the only default I typically change).  I'm fine with this as long as it's consistent.